axopy/pipeline/common.py

```diff
@@ -404,15 +404,40 @@ class Estimator(Block):
     estimator : object
         An object implementing the scikit-learn Estimator interface (i.e.
         implementing ``fit`` and ``predict`` methods).
+    return_proba : boolean, optional (default: False)
+        If True, return probability estimates.
```

Comment:
Just to make it 100% clear, could you change this to `If True, use the estimator's ``predict_proba`` method instead of ``predict`` to return probability estimates.` Likewise for `return_log_proba`.